tests/conftest.py

```diff
@@ -25,6 +32,18 @@ def pytest_collection_modifyitems(config, items):
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
 
+    # Limit the number of permutations of parametrised tests to run.
+    limit = config.getoption("--limit")
+    if limit > 0:
+        tests_by_name = {item.name: item for item in items}
+        # Add the name of parametrize base tests to this list.
+        tests_to_skip_parametrize = ["test_noise_order"]
```

Comment:
I don't think you need this if you actually tag the test itself with `@limit(N)` right?

Reply:
I was not able to get this to work. Maybe Im not tagging the test correctly but the few things I have tried did not work

Reply:
I do think I like this though having a list of all tests we do this for and not having to hunt for them.